components/Player/Pages/playerProsColumns.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Link } from 'react-router';
+import { transformations } from '../../../utility';
+import PercentContainer from '../../PercentContainer';
+
+export default [{
+  displayName: 'Player',
```

Comment:
You don't have to do this now, but I think the player cells should match the style that @blukai established for heroes.